pkg/vcs/github_client/message.go

```diff
@@ -27,6 +95,11 @@ func (c *Client) PostMessage(ctx context.Context, pr vcs.PullRequest, message st
 		message = message[:MaxCommentLength]
 	}
 
+	if err := c.deleteLatestRunningComment(ctx, pr); err != nil {
```

Comment:
Can you not delete this and instead update it with the first batch of the message? why: because it'll leave a lot of `comment deleted` trail on the github PR.

Reply:
Sorry, replied a bit lower - https://github.com/zapier/kubechecks/pull/301#discussion_r2060382442.
What do you think?